static/css/marble.css

```diff
@@ -651,30 +655,13 @@ h6{
 
 /*FAQ*/
 
-.margin-faq-content{
-    margin-top: 20px;
-}
-
 /*Extras*/
 
-/*Feedback*/
-
-.feedback-card{
```

Comment:
This class is still used in the html. Did you mean to remove this? Otherwise, remove the unused class in the html to reduce clutter

Reply:
This should be added back in too.